db/migrate/20241119131515_add_price_to_scooters.rb

```diff
@@ -0,0 +1,5 @@
+class AddPriceToScooters < ActiveRecord::Migration[7.1]
+  def change
+    add_column :scooters, :price_per_day, :float
```

Comment:
here you called it :price_per_day. If you want to do :price then do a migration and change it. if not just change the other places where I left a comment